compiler/rustc_codegen_llvm/src/llvm/ffi.rs

```diff
@@ -2586,8 +2559,6 @@ unsafe extern "C" {
 
     pub fn LLVMRustGetElementTypeArgIndex(CallSite: &Value) -> i32;
 
-    pub fn LLVMRustIsBitcode(ptr: *const u8, len: usize) -> bool;
```

Comment:
Historical note: This became unused in https://github.com/rust-lang/rust/commit/9de0d147f479f94c7cb49f1573e90fc529cda476.